cli/fabric.c

```diff
@@ -209,7 +219,7 @@ static int portcfg_set(int argc, char **argv)
 		DEVICE_OPTION,
 		{"phys_port_id", 'p', "NUM", CFG_INT,
 		 &cfg.phys_port_id, required_argument,
-		 "physical port id", .require_in_usage = 1},
+		 "Physical port ID", .require_in_usage = 1},
```

Comment:
In general, help text is not a sentence and is not capitalized. Most aren't so I wouldn't change it to this.

Reply:
I agree it would probably be better lowercase, but all other help text is capitalized on the first letter, so I did this to be consistent.

Reply:
Looking at this patch it looks like most of it is not capitalised... It's probably just same newer help texts that were added that are capitalised. I'd standardise on lower case.

Reply:
Ok so it looks like the top-level command descriptions are uppercase for main, fabric, "version", and "help", but not gas. The subcommand descriptions are capitalized for fabric but not main and gas. And subcommand choices (e.g. for evcntr-setup events) seem to all be capitalized. I'll change everything to lowercase. Does that sound good?

Reply:
Yes